rq/worker.py

```diff
@@ -227,7 +227,11 @@ def __init__(
         serializer=None,
     ):  # noqa
 
-        connection = self._set_connection(connection, default_worker_ttl)
+        self.default_result_ttl = default_result_ttl
+        self.default_worker_ttl = default_worker_ttl
```

Comment:
Can we rename `self.default_worker_ttl` to `self.worker_ttl`? This is the actual worker TTL that will be used.

`self.default_result_ttl` can be kept unchanged because it's indeed a "default", each job can specify it's own TTL.

Reply:
In version 2.0, we can also change the `default_worker_ttl` argument to `worker_ttl`